scraper/scraper.py

```diff
@@ -249,14 +263,21 @@ def fetch_centre_slots(
 def centre_iterator(platforms=None):  # pragma: no cover
     visited_centers_links = set()
     for center in ialternate(
-        ordoclic_centre_iterator(),
-        mapharma_centre_iterator(),
-        maiia_centre_iterator(),
-        avecmondoc_centre_iterator(),
-        doctolib_center_iterator(),
-        gouv_centre_iterator(),
+        # ordoclic_centre_iterator(),
```

Comment:
Tu as désactivé les autres itérateurs ici

Reply:
C'est corrigé